person.rb

```diff
@@ -1,19 +1,34 @@
+require_relative './nameable'
```

Comment:
- [ ] According to the project requirements, your Person class should inherit from the Nameable class, please update it.

Reply:
Thanks a million @omar-labana  for clarity - Fixed the issue as per guide.